alembic/env.py

```diff
@@ -43,6 +43,8 @@ def run_migrations_offline():
     context.configure(url=url)
 
     with context.begin_transaction():
+        if config.get_main_option('bdr').strip().lower() == 'true':
```

Comment:
It might be nice to make it possible for this option to not be present and default to False so upgrading users don't have to add this to their configs.